widget/embedded/src/lib/tabManager/tabManager.ts

```diff
@@ -0,0 +1,235 @@
+import type {
```

Comment:
We don't usually use camel case names for file names. can you double check it with other similar files?

Reply:
We've been using camel case for file names in many places already.